src/main/Cleanup.java

```diff
@@ -0,0 +1,61 @@
+package main;
+
+import com.microsoft.azure.management.netapp.v2019_11_01.implementation.AzureNetAppFilesManagementClientImpl;
+import rx.Observable;
+
+import java.util.concurrent.CompletableFuture;
+
+public class Cleanup {
+    /**
+     * Deletes destination replication connection
+     * @param anfClient Azure NetApp Files Management Client
+     * @param destinationResourceGroupName Resource Group name of the destination ANF
+     * @param destinationAnfAccountName Destination ANF account name
+     * @param destinationCapacityPoolName Destination ANF capacity pool name
+     * @param destinationVolumeName Destination ANF volume name
+     */
+    public static CompletableFuture<Observable<Void>> deleteReplicationOnDestinationAsync(AzureNetAppFilesManagementClientImpl anfClient, String destinationResourceGroupName, String destinationAnfAccountName, String destinationCapacityPoolName, String destinationVolumeName)
```

Comment:
I think this method should be named more generically, following all arguments and variables, if we can define all these settings, e.g. we don't need to say destinationXYZ,